pkg/proxy/winkernel/proxier.go

```diff
@@ -965,6 +969,23 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
+	hnsNetworkName := proxier.network.name
+	hns := proxier.hns
+
+	existingSourceVip, err := hns.getEndpointByIpAddress(proxier.sourceVip, hnsNetworkName)
+	if existingSourceVip == nil {
+		_, err = newSourceVIP(hns, hnsNetworkName, proxier.sourceVip, proxier.hostMac, proxier.nodeIP.String())
+	}
+	if _, ok := err.(hcn.NetworkNotFoundError); ok {
+		klog.V(4).Infof("Cannot find network %v", hnsNetworkName)
```

Comment:
These are critical error messages. User should be notified of such error, since this mean that something is wrong in the system, which shouldn't happen. There should also be a documented guidance somewhere, saying what to do when this happens. Can you come up with a better error code and message and see if this can be documented somewhere ?